.github/workflows/main.yml

```diff
@@ -74,7 +74,7 @@ jobs:
         source $CONDA/etc/profile.d/conda.sh
         pkg=local::anaconda-anon-usage
         conda install anaconda-client constructor conda-index $pkg
-        if [[ ! "${{ matrix.cversion }}" =~ 4.* ]]; then
+        if [[ "${{ matrix.cversion }}" == 23.7.* ]]; then

```

Comment:
the libmamba case was erroring out for 23.1.0 and 22.11.1. this started happening because I switched to doing `conda install` in the integration test instead of `conda search` for fuller testing, and libmamba was erroring out with this exception:
```
libmambapy.bindings.MambaNativeException: Cache not loaded
```
This seems to be caused by the fact I'm using a fake channel name and a fake package to ensure cache hits don't happen. Since later versions of conda+libmamba don't have this problem, and we're really just doing a sanity check to make sure the user agents are still returned in the libmamba case, I'm choosing to just run the libmamba tests for the newest version of conda.